src/java/org/apache/cassandra/cql3/conditions/ColumnCondition.java

```diff
@@ -546,6 +737,7 @@ private static boolean listAppliesTo(ListType<?> type, Iterator<Cell<?>> iter, L
 
         private static boolean setAppliesTo(SetType<?> type, Iterator<Cell<?>> iter, Set<ByteBuffer> elements, Operator operator)
         {
+            // This is redundant since it is already a sorted set?
```

Comment:
It certainly looks that way. `Value` always stores a `SortedSet` using the element type as a comparator. Probably worth streamlining.

Reply:
I removed the extra sort, I also removed the method containing the sort, and also removed the same wrapper method for list which didn't do much. LMK if it looks good.